site/content/en/docs/Getting Started/create-fleet.md

```diff
@@ -18,7 +18,7 @@ The following prerequisites are required to create a GameServer:
 
 If you don't have a Kubernetes cluster you can follow [these instructions]({{< ref "/docs/Installation/_index.md" >}}) to create a cluster on Google Kubernetes Engine (GKE), Minikube or Azure Kubernetes Service (AKS), and install Agones.
 
-For the purpose of this guide we're going to use the {{< ghlink href="examples/simple-udp/" >}}simple-udp{{< /ghlink >}} example as the GameServer container. This example is a very simple UDP server written in Go. Don't hesitate to look at the code of this example for more information.
+For the purpose of this guide we're going to use the {{< ghlink href="examples/simple-game-server/" >}}simple-game-server{{< /ghlink >}} example as the GameServer container. This example is a very simple UDP server written in Go. Don't hesitate to look at the code of this example for more information.
```

Comment:
The changes in this file should be changed with the next release.

Reply:
Then if that's true, it should be the entire getting started guide, not just this page.

But I would argue - if the entire guide still works, then we should be able to flip it whenever. The `simple-game-server` example has been around for several releases now and the images are available as well.